src/lib/modules/coverage/contract_source.js

```diff
@@ -336,8 +317,38 @@ class ContractSource {
         });
       });
     }
+  }
 
-    return coverage;
+  _buildContractBytecode(contractName, contractBytecode, opcodes, sourceMaps) {
+    var bytecodeMapping = contractBytecode[contractName];
+    var bytecodeIdx = 0;
+    var pc = 0;
+    var instructions = 0;
+    var previousSourceMap = null;
```

Comment:
I know those have mostly been moved around, but maybe this is a good opportunity to get rid off `var` and use at least `let` in here?